src/components/common/TmSessionExtension.vue

```diff
@@ -76,13 +76,18 @@ export default {
       })
     },
     async signInAndRedirect(account) {
+      let accountNetwork = await this.$store.dispatch("getNetworkByAccount", {
+        network: account.network,
+        address: account.address
+      })
+      if (!account.network && accountNetwork) {
+        account.network = accountNetwork.id
+      }
```

Comment:
This looks a bit strange no? How is it possible that there is no `account.network` but yes accountNetwork if you are assigning `account.network` inside `accountNetwork`?

Guess it would look like 
```
accountNetwork {
network: undefined,
address: "blablabla"
}
```
Or does it error? Did you test this?

Reply:
It tries to define a network by prefix